common/network-common/src/main/java/org/apache/spark/network/client/TransportClientFactory.java

```diff
@@ -173,9 +173,11 @@ public TransportClient createClient(String remoteHost, int remotePort)
     final InetSocketAddress resolvedAddress = new InetSocketAddress(remoteHost, remotePort);
     final long hostResolveTimeMs = (System.nanoTime() - preResolveHost) / 1000000;
     if (hostResolveTimeMs > 2000) {
-      logger.warn("DNS resolution for {} took {} ms", resolvedAddress, hostResolveTimeMs);
+      logger.warn("DNS resolution for {} took {} ms isUnresolved {}",
+              resolvedAddress, hostResolveTimeMs, resolvedAddress.isUnresolved());
```

Comment:
Do you have a sample output of this log entry?

Reply:
It is a boolean, so "... isUnresolved true|false"

Reply:
Nit: to be clearer this could be `isUnresolved=true` etc. Or could get fancier and make the message change to either "DNS resolution failed for ..." or "DNS resolution succeeded for .."

Reply:
I still think this log statement can be slightly improved if we're bothering.

Reply:
Fixed in code.